### ISSUE TITLE
650 rival not being detected Manjaro linux

### ISSUE THREAD

**Issue by elderwolf123** [libratbag/libratbag#1506]
=
- version 0.17 
- Distribution: Manjaro Linux 
- Linux 6.1.38-1-MANJARO x86_64 GNU/Linux

Device info (if applicable):
```
$ ratbagctl <device> info
ratbagctl steelseries-rival-650-wireless info                                                                                                                                                                               
Unable to find device steelseries-rival-650-wireless

**Describe the bug**
When trying to get the 650 rival working with piper its not being detected by libratbag and piper even though the device is meant to be supported

**To Reproduce**
Steps to reproduce the behavior:
1.  install libratbag and piper
2. launch piper
3. use ratbagctl to detect device

**Logs**
Please start the daemon with verbosity and reproduce the issue.

First make sure it isn't running already.
```
$ ratbagd --verbose=raw
OUTPUT HERE
ratbag debug: New device: SteelSeries SteelSeries Rival 650 Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1038:1726
ratbag debug: New device: SteelSeries SteelSeries Rival 650 Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1038:1726
ratbag debug: New device: SteelSeries SteelSeries Rival 650 Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1038:1726
ratbag debug: New device: SteelSeries SteelSeries Rival 650 Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 1038:1726
ratbag debug: New device: Alps Alps Touchpad
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 044e:1217
ratbag debug: New device: WCOM483C:00 056A:483C
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 056a:483c
DBus server ready

**Additional context**

**Comment by staticssleever668** [comment]
Hello! This is a duplicate of #1438.
TLDR: this is because of a faulty device compatibility check introduced in libratbag v0.17.
For now your options are either rolling back to v0.16, or compiling libratbag yourself (e.g. libratbag-git piper-git from AUR).
I hope I can make a release this month so the issues and this trickery can stop.
